prov/gni/include/gnix.h

```diff
@@ -466,6 +471,8 @@ struct gnix_fid_ep {
 	bool tx_enabled;
 	bool rx_enabled;
 	bool requires_lock;
+	int last_cached;
+	struct gnix_addr_cache_entry addr_cache[5];
```

Comment:
Arbitrarily picked. Smaller than a cache line and easily unrolled by the compiler. If we make this a tunable, compiler optimizations may be reduced as it may expect the value to change.